room/compiler-processing/build.gradle

```diff
@@ -41,6 +41,7 @@ dependencies {
     testImplementation(JUNIT)
     testImplementation(JSR250)
     testImplementation(KOTLIN_COMPILE_TESTING)
+    testImplementation(KOTLIN_KSP) // force exact KSP version
```

Comment:
Is this because otherwise we end up using the outdated KSP from kotlin compile testing?

Reply:
yep